lib/puppet/indirector/file_metadata/file.rb

```diff
@@ -4,19 +4,4 @@
 
 class Puppet::Indirector::FileMetadata::File < Puppet::Indirector::DirectFileServer
   desc "Retrieve file metadata directly from the local filesystem."
-
-  def find(request)
-    return unless data = super
-    data.collect(request.options[:source_permissions])
```

Comment:
Implementation removed because it's now redundant, collect is called by the DirectFileServer.